tests/crypto/test_cases/CMakeLists.txt

```diff
@@ -47,8 +47,8 @@ zephyr_sources_ifdef(CONFIG_MBEDTLS_SHA256_C        test_vectors_hmac.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_SHA256_C        test_vectors_sha_256.c)
 zephyr_sources_ifdef(CONFIG_MBEDTLS_SHA512_C        test_vectors_sha_512.c)
 
-# This is set differently based on backend (hw_sw sets first, hw or sw isolated sets the second)
-if(CONFIG_MBEDTLS_CIPHER_AES_256_CCM_C OR CONFIG_MBEDTLS_CCM_C)
+# This is set differently based on backend
+if(CONFIG_CC310_MBEDTLS_CCM_C OR CONFIG_VANILLA_MBEDTLS_CCM_C OR CONFIG_MBEDTLS_CCM_C)
```

Comment:
Since we are in source code, it would suffice with MBEDTLS_CCM_C (defined in nrf-config.h)

Reply:
8abda7e09d0c871e53353e762735ecb9426cd792